policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/web/PolicyController.java

```diff
@@ -93,4 +103,12 @@ public void setTypePolicyAsync(@RequestBody final SetTypePolicyRequest request)
         LOGGER.info("Invoking setTypePolicyAsync: {}", request);
         service.setTypePolicy(request.getType(), request.getPolicy());
     }
+
+    @EventListener(ApplicationReadyEvent.class)
+    public void initPostConstruct() {
+        // Add example Policies to the policy-service cache
+        policyConfig.getPolicies().stream()
+                .map(cacheWarmer -> cacheWarmer.policyWarm(userConfig.getUsers()))
+                .forEach(entry -> service.setResourcePolicy(entry.getKey(), entry.getValue()));
+    }
```

Comment:
add toString, hashCode and equals methods

Reply:
Do we need them? Are we ever going to print a Controller? or compare one Controller to another?
I'd leave them out to avoid someone trying to do something weird

Reply:
I don't think the hashcode and equals is required here

Reply:
toString added.

Reply:
Agreed